.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,46 @@
+# Proposed change
+
+<!--- 
+Please include a summary of the change and which issue is fixed in the form of fix #XX for auto-linking.
+
+- Include relevant motivation and context
+- List any dependencies that are required for this change.
+--->
+
+## Type of change
+
+<!--- Please delete options that are not relevant for your change --->
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## How has this change been tested?
+
+### Automated testing
+
+If your are running automated testing for this module, we would love to hear from you, and potentially integrate it to the module standard workflow.
+
+<!---
+Please describe the tests that you ran to verify your changes and provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
+--->
+
+### Manual testing
+
+If no automated testing is run, please ensure that at least the three steps below are passing without any error.
+
+- [ ] Running `terraform apply` on each examples of this module provision the intended resources without any error
```

Comment:
I would rephrase this as "Running `terraform apply` on each example provided with this module provisions the intended resource(s) without any errors."